app/javascript/stylesheets/news.scss

```diff
@@ -0,0 +1,47 @@
+/* Styles for form */
+
+.news-form {
+  background-color: $white;
+  padding: 20px;
+  margin: 20px auto;
+  border: 1px solid #C1BFC1;
+  border-radius: 20px;
+}
+
+.news-w95 {
+  width:95%;
+}
+
+.news-item {
+  width: 100%;
+}
+
+/* Styles for image preview */
+
+.news-img-prev {
+  text-align: center;
+  width: 100%;
+  border: 1px solid $black;
+  border-radius: 20px;
+}
+
+/* Styles for control */
+
+.news-control {
+  display: flex;
+  justify-content: space-around;
+}
+
+.button-width-120 {
+  width: 120px !important;
```

Comment:
try to avoid use of '!imortant', better to add this property to new class, and apply it in view